classes/02class/exercises/c02-network01/Faria-Marcio/c02-network01.txt

```diff
@@ -0,0 +1,7 @@
+IP			IP in binary				subnet in binary			subnet in decimals	is it private 	how many IP addresses?	
+																or public?	
+10.0.0.1/24		00001010.00000000.00000000.00000001	11111111.11111111.11111111.00000000	255.255.255.0		private		254	
+192.168.0.1/16		11000000.10101000.00000000.00000001	11111111.11111111.00000000.00000000	255.255.0.0		private		65534	
+249.165.166.135/30	11111001.10100101.10100110.10000111	11111111.11111111.11111111.11111100	255.255.255.252		public		2	
+236.68.223.18/32	11101100.01000100.11011111.00010010	11111111.11111111.11111111.11111111	255.255.255.255		public		0	

```

Comment:
For `236.68.223.18/32`, hosts = would be 1, but in AWS, the minimum subnet size is /28